.github/workflows/test.yaml

```diff
@@ -8,18 +8,55 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    env:
+      SRC_DIR: ${{ github.workspace }}
+      # https://github.com/axonweb3/axon/commits/forcerelay-dev
+      AXON_COMMIT: 343f329b78b8187e28fdac9e9af6c28222656b92
+      AXON_HTTP_RPC_URL: "http://localhost:8000"
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version: 18
           cache: 'yarn'
           cache-dependency-path: 'yarn.lock'
+
+      - name: Prepare Axon source
+        run: git clone --recursive https://github.com/axonweb3/axon.git $SRC_DIR/axon && cd $SRC_DIR/axon && git checkout $AXON_COMMIT
+
+      - name: Rust cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: "."
+
+      - name: Cache axon bin
+        id: cache-axon-bin
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-axon-bin
+        with:
+          path: ${{env.SRC_DIR}}/axon/target/release/axon
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ env.AXON_COMMIT }}
+  
+      - name: Build Axon
+        if: ${{ steps.cache-axon-bin.outputs.cache-hit != 'true' }}
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --manifest-path ${{env.SRC_DIR}}/axon/Cargo.toml
+      
+      - name: Add axon bin to path
+        run: echo "${{env.SRC_DIR}}/axon/target/release/" >> $GITHUB_PATH
+
+      - name: Config axon
+        run: axon init --config devtools/chain/config.toml --chain-spec devtools/chain/specs/single_node/chain-spec.toml
+
+      - name: Start axon
+        run: axon run --config devtools/chain/config.toml
+
       - run: yarn install
       - run: npx truffle version
 
-      - name: Run Ethereum locally for test
-        run: npx ganache > ganache.log &
       - run: yarn compile
       - name: Run test
-        run: npx truffle test
+        run: npx truffle test --network axon
```

Comment:
`--network axon` is not the local Axon network.
See https://github.com/wenyuanhust/ibc-solidity-contract/blob/172af680257a3c82ddc64aeffea9a80ae6c1efd0/truffle-config.js#L57-L65

Reply:
I have set AXON_HTTP_RPC_URL: "http://127.0.0.1:8000/"